docs/gh-actions/gh-update.md

```diff
@@ -15,44 +15,50 @@ This is only available with Loop 3 and Loop development branch.
     * 5 minutes to regenerate token
     * 5 minutes to update the GH_PAT Secret for each of your app repositories
 
-!!! abstract "Summary"
+??? abstract "`Page Summary` (click to open/close)"
     **You must build Loop every 90 days when you use the GitHub build method - this is a TestFlight requirement.**
```

Comment:
Use italics (or even bold when needed), like so `*Loop*` to prevent `Loop` from being translated literally.